### ISSUE TITLE
Burn / ¥$, Kanye West, Ty Dolla $ign

### ISSUE THREAD

**Issue by Ugnasijus1** [surfbryce/beautiful-lyrics#739]
### Before you list your Songs:

- [X] IF you are reporting multiple songs, add ", ..." to the end of your title.
- [X] For every song you want to report, check Musixmatch or Apple Music and check to see if the Lyrics are any Different than what Beautiful Lyrics has.
- [X] For every song you want to report, point out the Incorrect Lyrics and what the correct ones are (ONLY IF the Lyrics aren't entirely incorrect).

### List Song Links

[https://open.spotify.com/track/4E3XDKil4ElJt6sXuz5cc6?si=7df7526cd1f44c2e](https://open.spotify.com/track/4E3XDKil4ElJt6sXuz5cc6?si=7df7526cd1f44c2e) at 0:42 the lyrics are "when my campaign turned to cam-pain" when it should be "when my campaign turned to canned pain
![image](https://github.com/surfbryce/beautiful-lyrics/assets/102345753/c734457a-9a5f-4141-b896-fe9d5d2e3493)

**Comment by Cosmic1212** [comment]
yeah a lot of incorrect lyrics are there. i think it would be better to use genius, lrclib, or musixmatch for better lyrics
